### DISCUSSION TITLE
Making individual cells in table NoClip?

### DISCUSSION THREAD

**Discussion by sev-** [ocornut/imgui#7663] [New Users Build/Link/Run issues ONLY!]
I want certain columns in my table content not clipping, as shown on a picture below.

Is there such facility, because so far I saw `ImGuiTableColumnFlags_NoClip` applied only on the whole column in the header.

<img width="232" alt="Screenshot 2024-06-04 at 23 06 16" src="https://github.com/ocornut/imgui/assets/365188/fc87ffe2-c86d-4982-abf1-71d6a21da9ed">

**Comment by ocornut** [comment]
Please open an issue. Also see #3565
